transcom/mymove
# B 17991 add convert to customer api value

## [Agility ticket](tbd)
B-17991

## Summary

Is there anything you would like reviewers to give additional scrutiny?

This story adds a "convertToCustomerExpense" field to the 
**GET** move-task-orders/{moveTaskOrderID}/service-items/{mtoServiceItemID} endpoint.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. As a TOO select a move with an Origin SIT service item
2. Use developer tools to see the 'convertToCustomer' expense field is populated for each serviceItem listed

### Frontend


## Screenshots
<img width="464" alt="Screenshot 2023-12-26 at 7 16 46 AM" src="https://github.com/transcom/mymove/assets/146969726/49274cbc-7769-47d1-b49d-86baaca2bf05">
